sys/net/network_layer/ng_sixlowpan/frag/rbuf.c

```diff
@@ -204,7 +213,7 @@ static bool _rbuf_update_ints(rbuf_t *entry, uint16_t offset, size_t frag_size)
           new->start, new->end, ng_netif_addr_to_str(l2addr_str,
                   sizeof(l2addr_str), entry->src, entry->src_len));
     DEBUG("%s, %u, %u)\n", ng_netif_addr_to_str(l2addr_str,
-          sizeof(l2addr_str), entry->dst, entry->dst_len),
+            sizeof(l2addr_str), entry->dst, entry->dst_len),
```

Comment:
Those are parameters of `ng_netif_addr_to_str()` so that's why there was an indentation introduced here.